DevFAQ.md

```diff
@@ -0,0 +1,66 @@
+## Table of Contents:
```

Comment:
all the questions below are answered in other pages, why do we need the duplication?
let's instead improve whatever is needed in the other places, and if we want to keep this page as a table of contents then maybe let's have here questions and only links to the relevant documentation?

Reply:
@benr-ml - FAQs are a common section in developer oriented product docs, to allow someone to refer to key information easily, rather than have to go through detailed docs for every piece of information. This is good redundancy.

Reply:
I can keep it updated based on any major updates or new things, or if devs need any other info readily available that may be embedded in the detailed docs.

Reply:
These answers are very partial and devs should **never** depend on them for their actual apps - if we still want to leave it in (🤷 ), let's add a large disclaimer at the top of the page saying that those answers are partial and devs must read the full documentation before building any real application.